01-reading.Rmd

```diff
@@ -38,9 +38,10 @@ By the end of the chapter, students will be able to:
 - retrieve data from a database query and bring it into R using the `collect` function from the `dbplyr` package
 - use `write_csv` to save a data frame to a `.csv` file
 
-- (*optional*) scrape data from the web
-    - read/scrape data from an internet URL using the rvest `html_nodes` and `html_text` functions
-    - compare downloading tabular data from a plain text file (e.g. `.csv`) from the web versus scraping data from a `.html` file
+- (*optional*) obtain data using APIs and web scraping
```

Comment:
Is the first time we see the acronym API? If so, should we define it here, or should we do that in the main text? I am thinking here and in the main text...

Reply:
agree, both!